src/test/features/claimant-response/routes/check-and-send.ts

```diff
@@ -103,6 +103,122 @@ describe('Claimant response: check and send page', () => {
             .expect(res => expect(res).to.be.successful.withoutText('Court decision'))
         })
 
+        it('should render page when everything fine when Comp/Org as Defendant admission is accepted ' +
+          'but payment plan is rejected to be paid IMMEDIATELY', async () => {
+          claimStoreServiceMock.resolveRetrieveClaimByExternalId(claimStoreServiceMock.samplePartialAdmissionWithPaymentBySetDateCompanyData)
+          draftStoreServiceMock.resolveFind(draftType, {
+            courtDetermination: undefined,
+            alternatePaymentMethod: {
+              paymentOption: {
+                option: {
+                  value: 'IMMEDIATELY',
+                  displayValue: 'Immediately'
+                }
+              }
+            },
+            acceptPaymentMethod: {
+              accept: {
+                option: 'no'
+              }
+            },
+            settlementAgreement: {
+              signed: false
+            },
+            formaliseRepaymentPlan: {
+              option: {
+                value: 'referToJudge',
+                displayValue: 'Refer to judge'
+              }
+            }
+          })
+          draftStoreServiceMock.resolveFind('mediation')
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.successful.withText('Check your answers'))
+            .expect(res => expect(res).to.be.successful.withText('How would you like the defendant to pay'))
+            .expect(res => expect(res).to.be.successful.withText('Immediately'))
+            .expect(res => expect(res).to.be.successful.withoutText('Court decision'))
+        })
+
+        it('should render page when everything fine when Comp/Org as Defendant admission is accepted ' +
+          'but payment plan is rejected to be paid BY SET DATE', async () => {
+          claimStoreServiceMock.resolveRetrieveClaimByExternalId(claimStoreServiceMock.samplePartialAdmissionWithPaymentBySetDateCompanyData)
+          draftStoreServiceMock.resolveFind(draftType, {
+            courtDetermination: undefined,
+            acceptPaymentMethod: {
```

Comment:
Can we extract some common baseline for these inline objects and then just call out the relevant differences for each test?

Reply:
Agreed with @singh-surjeet to leave it as is